ios/FluentUI/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -14,7 +14,7 @@ class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
     }
 
     private struct Constants {
-        static let labelVerticalMarginForOneLine: CGFloat = 14
+        static let labelVerticalMarginForOneLine: CGFloat = 15
```

Comment:
This doesn't need to be like a multiple of 4?

Reply:
The total pixel height of the cell was originally 50px, this addition makes it 52px

Reply:
my math doesn't add up tho?

Reply:
The text area height is currently at 22, + vertical margin top and bottom = 52